Cannot find module 'convert-source-map'

<!--
  If you are creating a bug report, please include a link to a runnable
  reproduction in a Git repository
-->
```
  * Test suite failed to run
    Cannot find module 'convert-source-map'
      at Object.<anonymous> (../../../common/temp/node_modules/.artifactory.dc.teliacompany.net/vue-jest/3.0.4/a8ad1c11314ee989f37a031897ab9024/node_modules/vue-jest/lib/process.js:14:26)
      at Object.<anonymous> (../../../common/temp/node_modules/.artifactory.dc.teliacompany.net/vue-jest/3.0.4/a8ad1c11314ee989f37a031897ab9024/node_modules/vue-jest/vue-jest.js:1:17)
```

vue-jest's package.json doesn't include this package, yet it is being required in process.js.

This causes errors when you use pnpm which eliminates all phantom dependencies.

Sorry for not providing a repro, I was hoping the issue is clear enough without repro.

--- a/package.json
+++ b/package.json
@@ -70,6 +70,7 @@
     "@babel/plugin-transform-modules-commonjs": "^7.2.0",
     "@vue/component-compiler-utils": "^2.4.0",
     "chalk": "^2.1.0",
+    "convert-source-map": "^1.6.0",
     "extract-from-css": "^0.4.4",
     "source-map": "^0.5.6",
     "ts-jest": "^24.0.0"

